packages/cli/src/commands/worker.ts

```diff
@@ -172,6 +172,12 @@ export class Worker extends BaseCommand {
 
 		const server = http.createServer(app);
 
+		app.get('/healthz/db', async (_req, res) => {
+			return Db.connectionState.connected && Db.connectionState.migrated
+				? res.status(200).send({ status: 'ok' })
+				: res.status(500).send({ status: 'error' });
+		});
```

Comment:
Planning to extract out a healthcheck controller in the near future, when working on worker metrics.